src/components/UserUpdateGroups.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <v-card outlined class="subgroup" :disabled="!adminUser">
+    <h2>User Groups</h2>
+    <v-row no-gutters>
+      <v-col class="col-4">
+        <v-checkbox
+            hide-details="auto"
+            v-for="(group, index) in allGroups"
+            :id="'group-' + index"
+            v-model="selectedGroups"
+            :value="group"
+            :label="group.name"
+            :key="group.id"
+        ></v-checkbox>
+      </v-col>
+    </v-row>
+    <div class="my-6">
+      <v-btn id="save-user-groups" class="secondary"
+             medium v-on:click="updateUserGroups()"
+             v-if="adminUser">
+        Save User Groups
+      </v-btn>
+    </div>
+  </v-card>
+</template>
+
+<script>
+import UsersRepository from "@/api/UsersRepository";
+import GroupsRepository from "@/api/GroupsRepository";
+
+export default {
+  name: "UserUpdateGroups",
+  props: ['userId'],
+  data() {
+    return {
+      adminUser: false,
+      joinedGroups: [],
+      allGroups: [],
+      selectedGroups: []
+    };
+  },
+  async created() {
+    this.checkIfAdmin();
+    await this.getUserGroupsData();
+  },
+  methods: {
+    checkIfAdmin: function() {
+      if (this.$keycloak.tokenParsed.resource_access['user-management'] &&
+          this.$keycloak.tokenParsed.resource_access['user-management'].roles.includes('user-management-admin')) {
```

Comment:
Is this a brand new role added just for this feature? Does it needed to be added to the Jenkins client configuration?

Reply:
yes it is, though I imagine it could be used to limit access to other features within the application at a later time. 
The client config for Jenkins has been updated

Reply:
This is the only comment in the pull request that I really care about.